BSB_lan.ino

```diff
@@ -9005,7 +9053,9 @@ void printWifiStatus()
 void setup() {
   decodedTelegram.telegramDump = NULL;
   pinMode(EEPROM_ERASING_PIN, INPUT_PULLUP);  
+ #if defined(EEPROM_ERASING_GND_PIN)
```

Comment:
Why do you include this `#if`? EEPROM_ERASING_GND_PIN should always be defined, shouldn't it?

Reply:
I don't need this PIN. The button is connected to one GPIO only.

Reply:
Where do you connect the other end of the pin? Directly to GND? The idea was to have to adjacent pins so that users don't have to bridge across the board or install a button. A paperclip plugged into two adjacent buttons would be enough.

Reply:
Yes, the other pin is connected to GND.

Reply:
In order to match our explanations in the manual for the other systems, both pins should be defined and defined in such a way that they are adjacent to each other, assuming that the ESP32 can set one pin to OUTPUT in LOW state and another one to INPUT_PULLUP.

Reply:
From my point of view, all these Values (EEPROM_ERASING_PIN, EEPROM_ERASING_GND_PIN and LED_BUILTIN) should be defined in BBS_lan_config.h. Also a missing define for EEPROM_ERASING_GND_PIN should be supported,

Reply:
@dukess, what do you think?

Reply:
I would prefer using GND. For me, use of two adjacent pins was somewhat unobvious (due to lack of experience with hardware).

Reply:
The question for me was whether we should put EEPROM_ERASING_PIN etc. into _config.h or leave it in the main code. Personally, I'd like to keep _config.h as clean as possible. But we could do it in such a way that the main code sets some default values (as it currently does) unless overriding defines have been added to _config.h. Would both of you agree to that?